src/node_url.cc

```diff
@@ -233,6 +233,9 @@ void BindingData::Parse(const FunctionCallbackInfo<Value>& args) {
   CHECK_GE(args.Length(), 1);
   CHECK(args[0]->IsString());  // input
   // args[1] // base url
+  // args[2] // raise Exception
+
+  const bool raise_exception = args[2]->BooleanValue(args.GetIsolate());
```

Comment:
Checking for false is better since you dont need to pass true for every call. You just need to pass the extra variable on parse() function and avoid serialization

```suggestion
  const bool do_not_raise_exception = args[2]->IsTrue(isolate);
```

Reply:
won't it become double negative might affect readibility of this function? why not raise_exception and default to true, I will just have to pass false

Reply:
the goal was to not change the behavior when not passing anything but I'll resolve the comment for now and fix it later.